conda/install.py

```diff
@@ -306,12 +306,19 @@ def create_meta(prefix, dist, info_dir, extra_info):
 
 
 def mk_menus(prefix, files, remove=False):
-    if abspath(prefix) != abspath(sys.prefix):
-        # we currently only want to create menu items for packages
-        # in default environment
+    """
+    Create cross-platform menu items (e.g. Windows Start Menu)
+
+    Passes all menu config files %PREFIX%/Menu/*.json to ``menuinst.install``.
+    ``remove=True`` will remove the menu items.
+    """
+    exclude_envs = ('_build', '_test')  # Exclude all envs starting with...
+    if basename(abspath(prefix)).lower().startswith(exclude_envs):
```

Comment:
`_` also excludes `_app_own_environment_{package}`. That is actually the one I want! The other way would be only to whitelist such envs.